### DISCUSSION TITLE
How to give a useQuery hook of zodios my own query key?

### DISCUSSION THREAD

**Discussion by ronalterya** [ecyrbe/zodios#329] [Q&A]
Thanks :)

**Comment by ecyrbe** [comment]
You can't, it's abtracted away for you.
If you need to invalidate a key you can get back zodios queryKey from the query or outside from `getKeyByPath` or `getKeyByAlias` helper methods :  
https://www.zodios.org/docs/client/react#zodios-key-helpers
